src/addressable.md

```diff
@@ -0,0 +1,99 @@
+# Addressability, `@` and sharing
+
+Variables are called _shared_ if their address is taken.
+This could allow them to be modified concurrently.
+Hence, to prove the absence of data races, Gobra reasons with permissions.
+Gobra requires us to mark variables as shared explicitly,
+with the intention that the programmer is aware in which cases, permission based reasoning is employed.
+
+<!-- We distinguish _shared arrays_ and _shared structs_ from their _exclusive_ counterparts.
+Since if the address of an array or struct is never taken, we do not have to worry about data races, and reasoning about them is much easier. -->
+
+
+## Shared variables with `@`
+
+Variables are marked as _shared_ by appending the ampersand symbol to a variable name.
+For a variable `b` we either write `b /*@@@*/` using an inline annotation or equivalently `b@` in `.gobra` files.
+``` go
+func main() {
+	b /*@@@*/ := 1
+	// b@ := 1 // .gobra version
+    x := &b
```

Comment:
formatting is wrong, there might be problems with rendering later (same for other examples)